File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt the user for height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    // Do some loops
    for (int i = 0; i < height; i++)
    {
        for (int k = height - 1; k > i; k--)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

# reprompt the user for positive int
while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    # print spaces
    for j in range(height - 1, i, -1):
        print(" ", end="")
    # print hastags
    for k in range(0, i + 1, 1):
        print("#", end="")
    # print a new line
    print()